c-sharp-apps-shay-shalom-izhak/transportaion-Cargo-App/RunTranportaionApp2.cs

```diff
@@ -0,0 +1,301 @@
+            Console.WriteLine($"Ship Load result 1: {shipLoadResult1}"); // Expected: True
+            DisplayLoadedItems(testShip, "after loading the first item onto the Ship at Port 1");
+
+            bool shipLoadResult2 = testShip.Load(testItem2);
+            Console.WriteLine($"Ship Load result 2: {shipLoadResult2}"); // Expected: True
+            DisplayLoadedItems(testShip, "after loading the second item onto the Ship at Port 1");
+
+            // Load items onto the plane
+            bool planeLoadResult1 = testPlane.Load(testItem1);
+            Console.WriteLine($"Plane Load result 1: {planeLoadResult1}"); // Expected: True
+            DisplayLoadedItems(testPlane, "after loading the first item onto the Plane at Port 1");
+
+            bool planeLoadResult2 = testPlane.Load(testItem2);
+            Console.WriteLine($"Plane Load result 2: {planeLoadResult2}"); // Expected: True
+            DisplayLoadedItems(testPlane, "after loading the second item onto the Plane at Port 1");
+
+            // Attempt to load more items to reach the maximum items limit on the ship
+            Console.WriteLine("Attempting to load additional items onto the Ship:");
+
+            bool shipLoadResult3 = testShip.Load(testItem3);
+            DisplayLoadResult(shipLoadResult3, "Ship", "Item 3");
+
+            bool shipLoadResult4 = testShip.Load(testItem4);
+            DisplayLoadResult(shipLoadResult4, "Ship", "Item 4");
+
+            bool shipLoadResult5 = testShip.Load(testItem5);
+            DisplayLoadResult(shipLoadResult5, "Ship", "Item 5");
+
+            // This should fail as it exceeds the maxItems limit
+            bool shipLoadResult6 = testShip.Load(testItem6);
+            DisplayLoadResult(shipLoadResult6, "Ship", "Item 6");
+
+            // Load the chair onto the train
+            bool trainLoadResult3 = testTrain.Load(chairItem);
+            DisplayLoadResult(trainLoadResult3, "Train", "Chair Item");
+
+            // Load additional items onto the plane to test its limits
+            Console.WriteLine("Attempting to load additional items onto the Plane:");
+
+            bool planeLoadResult3 = testPlane.Load(testItem3);
+            DisplayLoadResult(planeLoadResult3, "Plane", "Item 3");
+
+            bool planeLoadResult4 = testPlane.Load(testItem4);
+            DisplayLoadResult(planeLoadResult4, "Plane", "Item 4");
+
+            bool planeLoadResult5 = testPlane.Load(testItem5);
+            DisplayLoadResult(planeLoadResult5, "Plane", "Item 5");
+
+            // Load the table onto the train
+            bool trainLoadResult4 = testTrain.Load(tableItem);
+            DisplayLoadResult(trainLoadResult4, "Train", "Table Item");
+
+            // Load the sofa onto the train
+            bool trainLoadResult5 = testTrain.Load(sofaItem);
+            DisplayLoadResult(trainLoadResult5, "Train", "Sofa Item");
+
+            // Travel to Port 2
+            testTrain.DistanceToNextPort = 1500;
```

Comment:
Try to add here this code and see how you get error. 
I believe you can suggest some idea's from here regarding design patterns

` Console.WriteLine("lingar - " + testTrain.GetPricingList());`